test/cypress/e2e/txDetails.spec.js

```diff
@@ -4,34 +4,44 @@ import urls from '../../constants/urls';
 import ss from '../../constants/selectors';
 import regex from '../../../src/utils/regex';
 
-const transferTxId = '12400920197376315040';
```

Comment:
You somehow mixed this file. Can you please roll back to the version from sprint branch?

Reply:
fixed